libs/ui-watt/src/lib/components/input/shared/watt-picker-base.ts

```diff
@@ -140,13 +140,13 @@ export abstract class WattPickerBase
 
     if (this.range) {
       this.setRangeValue(
-        value as WattRange,
+        (value ?? this.initialValue) as WattRange,
         this.startInput.nativeElement,
         this.endInput.nativeElement
       );
     } else {
       this.setSingleValue(
-        value as Exclude<WattPickerValue, WattRange>,
+        (value ?? this.initialValue) as Exclude<WattPickerValue, WattRange>,
```

Comment:
suggestion: nice change. Can we add a unit test verifying the behaviour in case someone decides to poke with for whatever reason it at some point.